tests/unit/test_repos_repo.py

```diff
@@ -1042,10 +1042,29 @@ def test_collaborators(self):
 
         self.session.get.assert_called_once_with(
             url_for('collaborators'),
-            params={'per_page': 100},
+            params={'affiliation': 'all', 'per_page': 100},
+            headers={}
+        )
+
+    def test_collaborators_valid_affiliation(self):
+        """
```

Comment:
I know we're not great at this elsewhere in the tests, but I'm hoping to get our doc-strings cleaned up soon. I tried to do so in the code itself. I'm going to target the tests next and it'd be good to do less work :wink: 

Could you substitute something like

```
"""Verify we can list collaborator with a valid affiliation.

Let's ensure that our affiliation validation works.
"""

Reply:
Probably a mistake on my end. I usually follow the same principle.